projects/fal/src/fal/toolkit/file/providers/fal.py

```diff
@@ -45,6 +45,9 @@ def _should_retry(exc: Exception) -> bool:
     if isinstance(exc, HTTPError) and exc.code in RETRY_CODES:
         return True
 
+    if isinstance(exc, URLError):
+        return True
+
```

Comment:
```suggestion
    if isinstance(exc, HTTPError) and exc.code in RETRY_CODES:
        return True
    elif type(exc) is URLError:
        # URLError is a base class for other errors,
        # but it can be raised directly
        return True

```